utils/json_config.py

```diff
@@ -57,7 +57,8 @@ def config(self) -> dict:
             exists.  The empty persistent dictionary otherwise.
 
         """
-        if self._config == {}:
+        if self._config == {} or self.reload:
```

Comment:
`self.reload` is unnecessary.  When the `self.filepath` is changed, it is a new configuration.  The correct procedure is to first `reset()` the configuration prior to setting the new filepath:

```python
config.reset()
config.filepath = new_filepath
```

and the configuration will reload.  This is not done automatically or the current configuration will be lost.  If the previous configuration it to bne retained, it should be saved prior to the `reset()`.

This should already be done in the GUI.  I don't know about the CLI.

Reply:
I will try it out in the Cli!